### DISCUSSION TITLE
How to end a Tabset section ?

### DISCUSSION THREAD

**Discussion by dattosma** [requarks/wiki#2882] [Help / Questions] [answered]
Hello,

Does someone know how to end a tabset section ?

````
## Results {.tabset}
### Tab One
### Tab Two

I want this text outside the tabset, but in wiki.js it will appear under 'Tab Two'
````
for example rmarkdown is using {-} 
````
## Results {.tabset}
### Tab One
### Tab Two
## {-}

Text outside the tabset
````

**Comment by NGPixel** [comment, marked as answer]
You need to start a new header (of equal or greater level to your tabset), e.g.:

```
## Results {.tabset}
### Tab One
### Tab Two

## New Section Name
I want this text outside the tabset, but in wiki.js it will appear under 'Tab Two'
```

**Reply by AVTPJ** [reply]
Please include this in the docs - https://docs.requarks.io/en/editors/markdown#content-tabs

**Comment by Worive** [comment]
There's also a tricky way: 
Start a new empty tabset at the end
```
## Results {.tabset}
### Tab One
### Tab Two

## End Tabset {.tabset}
Content outside of tabset without new title :)
```

**Reply by tim-elmer** [reply]
THANK YOU. I just spent way too long trying to figure out how to get a tabset followed by a paragraph with no header.
